src/client/automation/playback/click/index.js

```diff
@@ -222,21 +221,43 @@ export default class ClickAutomation {
         return focusAndSetSelection(elementForFocus, simulateFocus, this.caretPos);
     }
 
+    _getElementForClick (targetElement, topElement, targetElementParentNodes, topElementParentNodes) {
```

Comment:
I suppose we have too complicated naming. 
a) I don't know what does it mean: `this.eventState.clickElement`. It maybe `elementForClick`. But it's strange to call like `this._getElementForClick(elementForClick)`..
b) what the difference between `targetElement` and `topElement`? `targetElement` was a top element before mousedown too.

So, my proposals: 
rename `this.eventState.clickElement` -> `this.eventState.mouseDownElement`
rename `this.targetElementParentNodes` -> `this.mousedownElementParentNodes`

so, this function will be `_getElementForClick(mouseDownElement, topElement)`

You need not send `targetElementParentNodes` because you can use `this.mousedownElementParentNodes`.
And you need not send `topElementParentNodes`, you just call `domUtils.getParents(topElement)` here